CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ The general format is:
 
 ```
 
+# 4.3.0 - DATE (30/12/2022)
+
+### Added
+
+- Note added to README for use of authentication method and Bearer token
+
```

Comment:
CHANGELOG entries are not necessary for documentation updates :) LGTM otherwise!

```suggestion
```